src/applications/simple-forms/mock-alternate-header-21-0845/components/CustomHeader.jsx

```diff
@@ -54,12 +59,16 @@ const CustomHeader = ({ formData, formConfig, router, currentLocation }) => {
                 aria-hidden="true"
                 className="fas fa-arrow-left va-c-font-size--xs vads-u-margin-top--1 vads-u-margin-right--0p5 vads-u-color--gray-medium"
               />
-              {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-              <Link onClick={goBack} className="vads-u-margin-right--4">
```

Comment:
In this custom header it uses two links at the top for "Back" and "Exit form". This change makes them anchors rather than buttons which accessibility wants. In order to get the links to dynamically update, useEffect was used above